### PR TITLE
Allow admin to change order status; closes #108

### LINKED ISSUE #108: As an admin, on the order dashboard I can transition order to different status (cancel, mark as paid, mark as completed)

### PR DESCRIPTION
**Pull request by EmilyMB**
Adds new functionality on the admin dashboard:

As an admin, I can:
1. click link "cancel" to change orders which are currently "ordered" or "paid" to "cancelled"
2. click link "mark as paid" to change orders which are "ordered" to "paid"
3. click link to "mark as completed"  to change orders which are currently "paid" to "completed"
4. see the total count of orders by status.

### CODE REVIEW

File: app/controllers/admin_controller.rb [hunk 1]
```diff
@@ -2,9 +2,13 @@ class AdminController < ApplicationController
...
-      @orders = Order.all
+      @orders = all_orders
     else
       @orders = Order.where("status = ?", params[:status])
     end
   end
+
+  def all_orders
+    Order.all
+  end
```
Comment (mikedao) [inline app/controllers/admin_controller.rb]: Why this change? Is this a change where a scope would make more sense?

Comment (EmilyMB) [inline app/controllers/admin_controller.rb]: I removed it now. This was part of an abandoned attempt to refactor. :mount_fuji:

**Comment by mikedao** [comment]
:+1:
